package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-dev-webpack",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
Is this version bump intentional or accidentally merged?

Reply:
there are 7 additional commits as the PR is based on master, it should be properly rebased on release